hubconf.py

```diff
@@ -4,66 +4,77 @@
 import os
 from pathlib import Path
 from safetensors import safe_open
-
 import torch
 from inference import Mars5TTS, InferenceConfig
 
-ar_url = "https://github.com/Camb-ai/MARS5-TTS/releases/download/v0.3/mars5_en_checkpoints_ar-2000000.pt"
-nar_url = "https://github.com/Camb-ai/MARS5-TTS/releases/download/v0.3/mars5_en_checkpoints_nar-1980000.pt"
-
-ar_sf_url = "https://github.com/Camb-ai/MARS5-TTS/releases/download/v0.3/mars5_en_checkpoints_ar-2000000.safetensors"
-nar_sf_url = "https://github.com/Camb-ai/MARS5-TTS/releases/download/v0.3/mars5_en_checkpoints_nar-1980000.safetensors"
-
-def mars5_english(pretrained=True, progress=True, device=None, ckpt_format='safetensors',
-                  ar_path=None, nar_path=None) -> Mars5TTS:
-    """ Load mars5 english model on `device`, optionally show `progress`. """
-    if device is None: device = 'cuda' if torch.cuda.is_available() else 'cpu'
-
-    assert ckpt_format in ['safetensors', 'pt'], "checkpoint format must be 'safetensors' or 'pt'"
-    
-    logging.info(f"Using device: {device}")
-    if pretrained == False: raise AssertionError('Only pretrained model currently supported.')
-    logging.info("Loading AR checkpoint...")
-
-    if ar_path is None:
-        if ckpt_format == 'safetensors':
-            ar_ckpt = _load_safetensors_ckpt(ar_sf_url, progress=progress)
-        elif ckpt_format == 'pt':
-            ar_ckpt = torch.hub.load_state_dict_from_url(
-                ar_url, progress=progress, check_hash=False, map_location='cpu'
-            )
-    else: ar_ckpt = torch.load(str(ar_path), map_location='cpu')
-
-    logging.info("Loading NAR checkpoint...")
-    if nar_path is None:
-        if ckpt_format == 'safetensors':
-            nar_ckpt = _load_safetensors_ckpt(nar_sf_url, progress=progress)
-        elif ckpt_format == 'pt':
-            nar_ckpt = torch.hub.load_state_dict_from_url(
-                nar_url, progress=progress, check_hash=False, map_location='cpu'
-            )
-    else: nar_ckpt = torch.load(str(nar_path), map_location='cpu')
-    logging.info("Initializing modules...")
-    mars5 = Mars5TTS(ar_ckpt, nar_ckpt, device=device)
-    return mars5, InferenceConfig
 
+# Centralized checkpoint URLs for easy management and updates
+CHECKPOINT_URLS = {
+    "ar": "https://github.com/Camb-ai/MARS5-TTS/releases/download/v0.3/mars5_en_checkpoints_ar-2000000.pt",
+    "nar": "https://github.com/Camb-ai/MARS5-TTS/releases/download/v0.3/mars5_en_checkpoints_nar-1980000.pt",
+    "ar_sf": "https://github.com/Camb-ai/MARS5-TTS/releases/download/v0.3/mars5_en_checkpoints_ar-2000000.safetensors",
+    "nar_sf": "https://github.com/Camb-ai/MARS5-TTS/releases/download/v0.3/mars5_en_checkpoints_nar-1980000.safetensors"
```

Comment:
Pls update with latest urls from the hubconf.py (i.e. just rebase this branch on latest changes from master)